utils/generate_types/index.js

```diff
@@ -536,39 +543,39 @@ class TypesGenerator {
       documentation,
       doNotGenerate: new Set([
         ...coreDocumentation.classesArray.map(cls => cls.name).filter(name => !assertionClasses.has(name)),
-        'PlaywrightAssertions',
-        'Test',
         'Fixtures',
-        'TestOptions',
-        'TestConfig.use',
-        'TestProject.use',
         'GenericAssertions.any',
         'GenericAssertions.anything',
         'GenericAssertions.arrayContaining',
         'GenericAssertions.closeTo',
         'GenericAssertions.objectContaining',
         'GenericAssertions.stringContaining',
         'GenericAssertions.stringMatching',
+        'PlaywrightAssertions',
+        'Test',
+        'TestConfig.use',
```

Comment:
I think you can remove this and `TestProject.use` now.

Reply:
Done. Removed both.